### PR TITLE
Configure the project to run on Heroku

### PR DESCRIPTION
**Pull request by igorkruglyak**

### CODE REVIEW

File: mysite/settings.py [hunk 1]
```diff
@@ -107,3 +114,10 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+
+# Heroku: Update database configuration from $DATABASE_URL.
+
+# pylint: disable=invalid-name
+db_from_env = dj_database_url.config(conn_max_age=500)
+DATABASES['default'].update(db_from_env)
```
Comment (webknjaz) [inline mysite/settings.py]: This hackery isn't needed. You can just assign the config where you declare the databases.

File: mysite/settings.py [hunk 2]
```diff
@@ -107,3 +114,10 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+
+# Heroku: Update database configuration from $DATABASE_URL.
```
Comment (webknjaz) [inline mysite/settings.py]: How is this helpful?

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: gunicorn mysite.wsgi --log-file -
```
Comment (webknjaz) [inline Procfile:1]: How is this related to changing django app settings?

**Comment by webknjaz** [comment]
> Add files and changes

That's at least two distinct actions. Requiring more than one commit.

File: mysite/settings.py [hunk 4]
```diff
@@ -2,13 +2,14 @@
 
 import os
 
+
```
Comment (webknjaz) [inline mysite/settings.py]: This seems unrelated.

Comment (igorkruglyak) [reply mysite/settings.py]: I fixed it.

File: mysite/settings.py [hunk 5]
```diff
@@ -25,6 +25,7 @@
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```
Comment (webknjaz) [inline mysite/settings.py:28]: Does the tutorial require you to use middleware in exactly this place? This suggests that you should modify the WSGI module: https://tutorial-extensions.djangogirls.org/en/heroku/.
WDYT?

File: requirements.txt [hunk 6]
```diff
@@ -1 +1,5 @@
+dj-database-url==0.5.0
```
Comment (webknjaz) [inline requirements.txt]: Since you don't use it in this PR, it's unnecessary.

**Comment by webknjaz** [comment]
ping

File: mysite/settings.py (hunk 5 above)
Comment (igorkruglyak) [reply mysite/settings.py:28]: I changed wsgi. When trying to do commit I get an error pylint: mysite/wsgi.py:6:0: E0611: No name 'DjangoWhiteNoise' in module 'whitenoise.django' (no-name-in-module)

**Comment by webknjaz** [comment]
ping

File: mysite/settings.py [hunk 7]
```diff
@@ -59,8 +60,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'djangogirls',
```
Comment (webknjaz) [inline mysite/settings.py:64]: dsfds

File: mysite/settings.py [hunk 8]
```diff
@@ -107,3 +112,7 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+# Add compression and caching support
+
```
Comment (webknjaz) [inline mysite/settings.py:117]: asfd

**Review by webknjaz — COMMENTED** [review]
test

File: requirements.txt [hunk 9]
```diff
@@ -1,3 +1,3 @@
 Django~=2.2.2
-psycopg2==2.8.3
+gunicorn==19.9.0psycopg2==2.8.3
```
Comment (webknjaz) [inline requirements.txt]: oops

Comment (igorkruglyak) [reply requirements.txt]: yep

**Review by webknjaz — APPROVED** [review]
